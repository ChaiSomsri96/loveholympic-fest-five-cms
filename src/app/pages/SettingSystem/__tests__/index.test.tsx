import * as React from 'react';
import { render } from '@testing-library/react';

import { SettingSystem } from '..';

jest.mock('react-i18next', () => ({
  useTranslation: () => {
    return {
      t: str => str,
      i18n: {
        changeLanguage: () => new Promise(() => {}),
      },
    };
  },
}));

describe('<SettingSystem  />', () => {
  it('should match snapshot', () => {
    const loadingIndicator = render(<SettingSystem />);
    expect(loadingIndicator.container.firstChild).toMatchSnapshot();
  });
});
