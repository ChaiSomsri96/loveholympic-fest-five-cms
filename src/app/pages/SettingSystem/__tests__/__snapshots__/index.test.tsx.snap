// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SettingSystem  /> should match snapshot 1`] = `
.c0 {
  padding: 2rem 1rem;
  background-color: #ecf0f5;
  height: 100%;
}

.c1 {
  color: #2c5282;
  font-family: Roboto;
  font-size: 24px;
  font-weight: bold;
}

<div
  class="c0"
>
  <div
    class="row"
  >
    <div
      class="col-sm-12"
    >
      <p
        class="c1"
      >
        Setting System
      </p>
    </div>
    <div
      class="col-12 col-sm-6"
    >
      <div
        class="card"
      >
        <div
          class="card-header"
        >
          URL Lohol Goods
        </div>
        <div
          class="card-body"
        >
          <div
            class="row"
          >
            <div
              class="pt-3 col-sm-12"
            >
              <input
                class="form-control"
                placeholder="loholGoodsURL"
                type="text"
                value=""
              />
            </div>
            <div
              class="pt-3 col-sm-12"
            >
              <input
                class="form-control"
                placeholder="youtubeHistoryURL"
                type="text"
                value=""
              />
            </div>
          </div>
          <button
            class="mt-4 btn btn-primary"
            type="button"
          >
            Save
          </button>
        </div>
      </div>
    </div>
    <div
      class="col-12 col-sm-6"
    >
      <div
        class="card"
      >
        <div
          class="card-header"
        >
          Logo System
        </div>
        <div
          class="card-body"
        >
          <button
            class="mt-4 btn btn-primary"
            type="button"
          >
            Save
          </button>
        </div>
      </div>
    </div>
  </div>
</div>
`;
